cyberessentials/cyberessentials.lua

```diff
@@ -4,7 +4,7 @@ local headingCalculator = require "cyberessentials/tools/headingcalculator"
 local repoTools = require "cyberessentials/tools/repo"
 
 
-local game = cyberpunk:new().game
+local game = cyberpunk:new({ game = Game }).game

```

Comment:
I haven't found a good solution for this, but the code doesn't run locally with this line change. It'll run with the game but I intend to be able to test the core functionality of the mod without the dependency of Cyberpunk.

Reply:
That was just a remnant of me working with the wrong base file, honestly hadn't even noticed you had removed { game = Game}